Mathlib/LinearAlgebra/Matrix/Spectrum.lean

```diff
@@ -148,6 +148,19 @@ lemma spectral_theorem' :
   simpa [ ← Matrix.mul_assoc, hA.eigenvectorMatrix_mul_inv, Matrix.one_mul] using
     congr_arg (hA.eigenvectorMatrix * ·) hA.spectral_theorem
 
+
+@[simp]
+theorem rank_unitary_mul {m n R : Type*} [Fintype n] [CommRing R] [DecidableEq n] [StarRing R]
```

Comment:
I understand that these can't go into an earlier file for import reasons, but they don't belong this far down the file. Please make a short section at the beginning of the file for these two.

Reply:
Right at the beginning?

Reply:
I created a section for this near the beginning.